common/lib/capa/capa/responsetypes.py

```diff
@@ -189,7 +189,10 @@ def __init__(self, xml, inputfields, context, system, capa_module):
             raise LoncapaProblemError(msg)
 
         for prop in self.required_attributes:
-            if not xml.get(prop):
+            prop_value = xml.get(prop)
+            if prop_value and isinstance(prop_value, str):
+                prop_value = prop_value.strip()
+            if not prop_value:
```

Comment:
we will not get anything other than `bytes`(type str) or `unicode`(type unicode), so `strip()` will work.